docs/build-framework/switches/target.md

```
@@ -0,0 +1,50 @@
+---
+seo_title: "Armbian build target switches: BOARD, BRANCH, RELEASE"
+description: "Select what the Armbian build framework builds: BOARD, kernel BRANCH (vendor, legacy, current, edge) and the Debian or Ubuntu RELEASE."
+---
+
+# Target selection
+
+Which board, kernel branch and userspace release to build. Set these to skip the interactive dialog.
+
+#### BOARD
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h4

(MD001, heading-increment)